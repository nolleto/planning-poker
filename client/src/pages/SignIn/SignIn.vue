<template>
  <SignInForm />
</template>

<script>
import SignInForm from '../../containers/SignInForm'

export default {
  name: 'SignIn',

  components: { SignInForm }
}
</script>

<style>

</style>
