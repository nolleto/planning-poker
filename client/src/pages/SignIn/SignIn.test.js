import SignIn from './index'
import { shallowMount } from '@vue/test-utils'

const shallowComponent = () => {
  return shallowMount(SignIn)
}

describe('SignIn component', () => {
  let wrapper

  describe('Snapshots', () => {
    beforeAll(() => {
      wrapper = shallowComponent()
    })

    it('matches', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
