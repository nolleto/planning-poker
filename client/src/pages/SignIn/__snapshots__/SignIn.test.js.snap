// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn component Snapshots matches 1`] = `
<div
  class="wrapper"
>
  <signinform-stub
    class="form"
  />
</div>
`;
